docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -87,6 +177,19 @@ The flag to enable this bootstrapping when starting the kubelet is:
 --experimental-bootstrap-kubeconfig="/path/to/bootstrap/kubeconfig"
 ```
 
+Additionally, in 1.7 the kubelet implements __alpha__ features for enabling rotation of both its client and/or serving certs.
+These can be enabled through the respective `RotateKubeletClientCertificate` and `RotateKubeletServerCertificate` feature
+flags on the kubelet, but may change in backward incompatible ways in future releases.
```

Comment:
What CSR logic is in the API server? I would have expected that in the controller manager. Does the controller manager detect this based on the API server configuration?